### PR TITLE
7.0.2 version update

### PR DESCRIPTION
**Pull request by Verasl**
### Purpose of this PR
Updating 7.0.x branch to next patch version

### CODE REVIEW

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,18 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.0.2] - 2019-08-15
+
 ## [7.0.1] - 2019-07-25
+### Changed
+- Platform checks now provide more helpful feedback about supported features in the Inspectors.
+
+### Fixed
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/CHANGELOG.md]: Are we allowed to patch this?

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 2]
```diff
@@ -4,15 +4,18 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.0.2] - 2019-08-15
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/CHANGELOG.md]: Either update data for all packages or none.

File: com.unity.render-pipelines.high-definition-config/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.0.2] - 2019-XX-XX
```
Comment (phi-lira) [inline com.unity.render-pipelines.high-definition-config/CHANGELOG.md]: Seems you forgot to update the high-definition-config.json?

File: com.unity.render-pipelines.universal/CHANGELOG.md (hunk 1 above)
Comment (Verasl) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: This is a question for @Unity-Technologies/gfx-docs

Comment (CatherineUnity) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: Yes, looks fine.

**Pull request merged**